pkgs/top-level/beam-packages.nix

```diff
@@ -58,7 +58,7 @@ rec {
     # `beam.packages.erlangR19.elixir`.
     inherit (packages.erlang) elixir elixir_1_5_rc elixir_1_4 elixir_1_3;
 
-    lfe = packages.erlang.lfe;
+    inherit (packages.erlang) lfe lfe_1_2;
```

Comment:
You need to re-inherit it in `pkgs/top-level/all-packages.nix` if you want it to be available at top level (I think it should be there)